Mathlib/Mathport/Syntax.lean

```diff
@@ -0,0 +1,727 @@
+syntax "destruct " term : tactic
+syntax (name := cases') "cases' " casesTarget,+ (" using " ident)?
+  (" with " (colGt binderIdent)+)? : tactic
+syntax (name := casesM) "casesM" "*"? ppSpace term,* : tactic
+syntax (name := casesType) "casesType" "*"? ppSpace ident* : tactic
+syntax (name := casesType!) "casesType!" "*"? ppSpace ident* : tactic
+syntax (name := «sorry») "sorry" : tactic
+syntax (name := iterate) "iterate" (num)? ppSpace tacticSeq : tactic
+syntax (name := repeat') "repeat' " tacticSeq : tactic
+syntax (name := abstract) "abstract" (ppSpace ident)? ppSpace tacticSeq : tactic
+syntax (name := have'') "have " Term.haveIdLhs : tactic
+syntax (name := let'') "let " Term.haveIdLhs : tactic
+syntax (name := suffices') "suffices " Term.haveIdLhs : tactic
+syntax (name := trace) "trace " term : tactic
+syntax (name := existsi) "exists " term,* : tactic
+syntax (name := eConstructor) "econstructor" : tactic
+syntax (name := left) "left" : tactic
+syntax (name := right) "right" : tactic
+syntax (name := constructorM) "constructorM" "*"? ppSpace term,* : tactic
+syntax (name := exFalso) "exFalso" : tactic
+syntax (name := injections') "injections" (" with " (colGt (ident <|> "_"))+)? : tactic
+syntax (name := simp') "simp'" "*"? (" (" &"config" " := " term ")")? (&" only")?
+  (" [" simpArg,* "]")? (" with " (colGt ident)+)? (ppSpace location)? : tactic
+syntax (name := simpIntro) "simpIntro" (" (" &"config" " := " term ")")?
+  (ppSpace (colGt (ident <|> "_")))* (&" only")? (" [" simpArg,* "]")? (" with " ident+)? : tactic
+syntax (name := dsimp) "dsimp" (" (" &"config" " := " term ")")? (&" only")?
+  (" [" simpArg,* "]")? (" with " (colGt ident)+)? (ppSpace location)? : tactic
+syntax (name := symm) "symm" : tactic
+syntax (name := trans) "trans" (ppSpace (colGt term))? : tactic
+syntax (name := acRfl) "acRfl" : tactic
+syntax (name := cc) "cc" : tactic
+syntax (name := substVars) "substVars" : tactic
+syntax (name := dUnfold) "dunfold" (" (" &"config" " := " term ")")?
+  (ppSpace (colGt ident))* (ppSpace location)? : tactic
+syntax (name := delta') "delta'" (colGt ident)* (ppSpace location)? : tactic
+syntax (name := unfoldProjs) "unfoldProjs" (" (" &"config" " := " term ")")?
+  (ppSpace location)? : tactic
+syntax (name := unfold) "unfold" (" (" &"config" " := " term ")")?
+  (ppSpace (colGt ident))* (ppSpace location)? : tactic
+syntax (name := unfold1) "unfold1" (" (" &"config" " := " term ")")?
+  (ppSpace (colGt ident))* (ppSpace location)? : tactic
+syntax (name := inferOptParam) "inferOptParam" : tactic
+syntax (name := inferAutoParam) "inferAutoParam" : tactic
+syntax (name := guardExprEq) "guardExpr " term:51 " =ₐ " term : tactic
+syntax (name := guardTarget) "guardTarget" " =ₐ " term : tactic
+syntax (name := guardHyp) "guardHyp " ident
+  ((" : " <|> " :ₐ ") term)? ((" := " <|> " :=ₐ ") term)? : tactic
+syntax (name := matchTarget) "matchTarget " term : tactic
```

Comment:
Already in Tactic.Basic.

Reply:
@digama0, could you explain your thinking about the `=ₐ` here? It seems contrary to the principle of preserving mathlib3 syntax.

Reply:
Done.